sbi/inference/posteriors/direct_posterior.py

```diff
@@ -283,6 +284,8 @@ def map(
                 (thus, the default is `10`.)
             show_progress_bars: Whether or not to show a progressbar for sampling from
                 the posterior.
+            force_update: Whether or not to re-calculate the MAP when x is unchanged and
```

Comment:
maybe remove the "or not" to make clear that `True` means that we do force update.